health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CloneCampaignWrapper.js

```
@@ -125,113 +125,118 @@ const CloneCampaignWrapper = (props) => {
+          className={"copy-campaign-popup"}
+          onOverlayClick={() => props.setCampaignCopying(false)}
+          onClose={() => props.setCampaignCopying(false)}
+          footerChildren={[
+            <Button
+              className={"campaign-type-alert-button"}
+              type={"button"}
+              size={"large"}
+              variation={"primary"}
+              label={t("SUBMIT")}
+              onClick={onNextClick}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)